ipwb/indexer.py

```diff
@@ -52,7 +52,11 @@
 
 DEBUG = False
 
-IPFS_API = ipfsapi.Client(ipwbUtils.IPFSAPI_MUTLIADDRESS)
```

Comment:
#DRY!!! Define a `createIPFSClient()` function in the `util.py` file that validates the address. If validation fails (i.e., an exception in thrown), return `None`, otherwise return the client object. In `indexer.py` and `replay.py` files call that function and assign the returned value to a variable. Then check if the variable is `None`, immediately exit. Reuse the same function inside the `isDaemonAlive()` function, but do not exit there, simply return `False` or whatever makes sense in that context.